massgov/mayflower
# DP-19425 Fix formatting of COVID global nav link on search.mass.gov

## Description
This PR includes some conditional logic to look for the keyword "covid" in the top-level menu item text, and add the class cv-alternative-style when detected. This ticket was created to address the missing COVID-19 styling in massgov-search, which pulls in its templates and styles from Mayflower React.

## Related Issue / Ticket

- [DP-19425](https://jira.mass.gov/browse/DP-19425)

## Steps to Test
**Testing in Mayflower**
1. Pull down this branch of Mayflower
2. Assuming other general setup steps are completed (see react/README.md), run `npm run build` from the react directory.
3. Run `npm run start`
4. Go to http://localhost:6006/?path=/story/molecules--mainnav.
5. Change the name of one of the links to "COVID-19" to see styling come in as expected.

**Testing in Massgov-search**
1. Pull down the develop branch
2. Follow normal set up steps to start the application as found in the README.
3. Switch back over to the Mayflower project and go to the react directory
4. Run the command `npm link`
5. Switch back to the Massgov-search project, staying in the root directory
6. Run `npm link @massds/mayflower-react`
7. Run `npm rebuild node-sass`
8. Run `npm run start`
9. Visit the application (likely at http://localhost:3000) and observe the COVID-19 link

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
